pkg/apis/ark/v1/register.go

```diff
@@ -41,27 +41,32 @@ func Resource(resource string) schema.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
+type typeInfo struct {
+	ItemType     runtime.Object
+	ItemListType runtime.Object
+}
+
+// CustomResources returns a map of all custom resources within the Ark
+// API group, keyed on Kind.
+func CustomResources() map[string]typeInfo {
```

Comment:
Would it make sense to re-use https://github.com/heptio/ark/blob/master/pkg/install/crd.go here? That way we'd have them defined in one place.

Reply:
I'd rather go the other way around (have `pkg/install` use the `pkg/apis/ark/v1` func), but yeah good point, I'll update the install package.

Reply:
Yeah, that direction does make more sense. If we use one common source then we're less likely to miss requiring any new CRDs getting added.

Reply:
updated.